packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -298,15 +315,32 @@ class _CodeViewState extends State<CodeView>
                         Expanded(
                           child: LayoutBuilder(
                             builder: (context, constraints) {
-                              return Lines(
-                                constraints: constraints,
-                                scrollController: textController,
-                                lines: lines,
-                                pausedFrame: pausedFrame,
-                                searchMatchesNotifier:
-                                    widget.controller.searchMatches,
-                                activeSearchMatchNotifier:
-                                    widget.controller.activeSearchMatch,
+                              final num fileWidth = lines.fold(
+                                  0,
+                                  (widestWidth, line) => math.max(widestWidth,
+                                      CodeView.calculateLineWidth(line)));
```

Comment:
measuring every single line in a large file will be fairly slow. You will probably notice a slowdown from this code given a 5000+ line file when you resize the window. 
Instead, take advantage of the fact that the font is fixed width and measure a single character and multiply or only measure the length of the longest line.

Reply:
Good point, now find the longest line and then measuring that